compiler/codegen/OMRAheadOfTimeCompile.cpp

```diff
@@ -115,6 +116,30 @@ void OMR::AheadOfTimeCompile::traceRelocationOffsets(uint8_t *&cursor, int32_t o
       }
    }
 
+
+#ifndef J9_PROJECT_SPECIFIC
+uint8_t* initializeAOTRelocationHeader(TR::IteratedExternalRelocation *relocation){
+   //This should not happen
+   return 0;
+}
+
+
+void processRelocations(){
+    //This Should not happen
+    }
+uint8_t *
+OMR::AheadOfTimeCompile::initializeCommonAOTRelocationHeader(TR::IteratedExternalRelocation *relocation, TR::RelocationRecord *reloRecord)
+   {
+   uint8_t *cursor = relocation->getRelocationData();
+   TR::Compilation *comp = TR::comp();
```

Comment:
Just confirming this is the only place to get the Compilation object from.  `TR::comp()` is expensive.

Reply:
It should be gone, I have a cached version in self->comp(). Thank you! Will be shipped in the commit after next

Reply:
Fixed that!